github_login.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Log in to GitHub</title>
+    <style>
+    body>section{
+        font-family:"Arial-black";
+        color: grey;
+    }
+    .serviceAgreement{
+        display: inline;
+    }
+    a{
+        text-decoration: none;
+    }
+    h3{
+        font-family:"Arial";
+        font-size: 80%;
+        text-align:start;
+        position:relative;
+        left:25px;
+    }
+    button{
+        font-family:"Arial";
+        font-size: 110%;
+        color: white;
+        background-color: green;
+        height: 70px;
+        width: 350px;
+        border-radius: 7px;
+        border: none;
+    }
+    input{
+        height: 30px;
+        width: 350px;
+        font-size: 110%;
+        border-style: solid;
+        border-radius: 7px;
+        border-color: lightgrey;
+    }
+    form{
+        border-style: solid;
+        border-radius: 7px;
+        border-width: 2px 2px 2px 2px;
```

Comment:
You can shorten this line by just writing border-width: 2px; because that tells the browser the border width is the same across all sides.